### PR TITLE
[EWL-6126/EWL-6172] Social Share Icon Updates

### PR DESCRIPTION
**Pull request by JaeBea**
## Ticket(s)
**Jira Ticket**
- [EWL-6126 | Social share options should be Facebook, Google Plus, Twitter, LinkedIn, Email](https://issues.ama-assn.org/browse/EWL-6126)
- [EWL-6172 | Social icons - request to change hover color](https://issues.ama-assn.org/browse/EWL-6172)

## Description
This work does the following:
- Add email social share icon svg
- Update all social share hover icons to use purple versions provided by UX
- Update all SG2 instances of youtube share link with email share link in mastheads.

### Dependencies
[EWL-6126: Social share email option #978](https://github.com/AmericanMedicalAssociation/ama-d8/pull/978)

## To Test
- Pull `feature/EWL-6126-social-share-options` branch
- Run `gulp serve` in ama-style-guide-2/styleguide to ensure a local version of style guide is running.
- This work is not dependent on D8, but affects the icon hover images and thus should be checked for consistency. Please test against [EWL-6126: Social share email option #978](https://github.com/AmericanMedicalAssociation/ama-d8/pull/978). Update ama-d8 branch and local provision vars accordingly. 
- Go to various page mastheads and confirm hover icons are lavender purple.
- Confirm mastheads no longer show youtube as a social share option.
- Test this in IE 11, Safari, and Firefox to ensure that with updates everything works same as before.

## Visual Regressions
A report is available [here](http://htmlpreview.github.io/?https://github.com/AmericanMedicalAssociation/ama-style-guide-2/blob/visual-regression-testing-artifact/feature/EWL-6126-social-share-options/html_report/index.html).

## Relevant Screenshots/GIFs
Check JIRA ticket for icons that have been replaced.

## Remaining Tasks
N/A

## Additional Notes
N/A

### CODE REVIEW

**Comment by designerneil** [comment]
@JaeBea can you run `gulp minifyIcons` to compress the icons
and `gulp svg2twig` to convert the svg into twig usable files?

File: styleguide/source/_patterns/01-atoms/media/icons/svg/icon-email.twig [hunk 1]
```diff
@@ -0,0 +1 @@
+<svg id="Layer_1" data-name="Layer 1" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 155.4 155.4"><defs><style>.cls-1{fill:#fff;stroke:#231f20;}.cls-1,.cls-2{stroke-miterlimit:10;}.cls-2{fill:none;stroke:#000;stroke-width:5px;}</style></defs><title>Email_default</title><circle cx="77.7" cy="77.7" r="77.7"/><rect class="cls-1" x="21.01" y="49.52" width="113.38" height="60.6"/><line class="cls-2" x1="17.47" y1="48.54" x2="79.83" y2="84.55"/><line class="cls-2" x1="141.14" y1="48.45" x2="77.37" y2="84.64"/></svg>
```
Comment (designerneil) [inline styleguide/source/_patterns/01-atoms/media/icons/svg/icon-email.twig]: need End of line break here

File: styleguide/source/_patterns/01-atoms/media/icons/svg/icon-facebook-hover.twig [hunk 2]
```diff
@@ -1 +1,24 @@
...
+                            <use xlink:href="#path-1"></use>
+                        </mask>
+                        <g id="Clip-2"></g>
+                        <path d="M36.1162,17.5937 L33.0182,17.5937 C30.5922,17.5937 30.1282,18.7417 30.1282,20.4337 L30.1282,24.1637 L35.9102,24.1637 L35.1612,30.0107 L30.1282,30.0107 L30.1282,45.0227 L24.0882,45.0227 L24.0882,30.0237 L19.0402,30.0237 L19.0402,24.1767 L24.0882,24.1767 L24.0882,19.8647 C24.0882,14.8577 27.1462,12.1207 31.5982,12.1207 C33.1032,12.1157 34.6072,12.1957 36.1042,12.3537 L36.1042,17.5797 L36.1162,17.5937 Z M29.0682,0.1817 C13.3862,0.1817 0.6732,12.8957 0.6732,28.5777 C0.6732,44.2607 13.3862,56.9747 29.0682,56.9747 C44.7522,56.9747 57.4662,44.2607 57.4662,28.5777 C57.4662,12.8957 44.7522,0.1817 29.0682,0.1817 Z" id="Fill-1" fill="#9073A6" mask="url(#mask-2)"></path>
+                    </g>
+                </g>
+            </g>
+        </g>
+    </g>
+</svg>
```
Comment (designerneil) [inline styleguide/source/_patterns/01-atoms/media/icons/svg/icon-facebook-hover.twig]: EOL needed

**Review by damontgomery — APPROVED** [review]
Edit: It looks like Neil was reviewing too and has some suggestions.

Previous review:

This works as described.

This was tested using the style guide as well as with the Drupal 8 local site using the related PR, https://github.com/AmericanMedicalAssociation/ama-d8/pull/978. For both, I looked at an article page and viewed the icons.

**Comment by JaeBea** [comment]
@damontgomery and @designerneil ,
I have run requested processes. This should have also fixed the end of line problems on the mentioned svg files.

**Review by designerneil — APPROVED** [review]

**Pull request merged**